resources/js/pages/game/[game]/requests.tsx

```diff
@@ -0,0 +1,26 @@
+import { useTranslation } from 'react-i18next';
+
+import { SEO } from '@/common/components/SEO';
+import { AppLayout } from '@/common/layouts/AppLayout';
+import type { AppPage } from '@/common/models';
+import { GameSetRequests } from '@/features/games/components/GameSetRequests';
+
+const SetRequestorsPage: AppPage = () => {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <SEO title={t('List of Set Requests')} description="A list of set requests for this game." />
```

Comment:
Consistency: take a peek at _resources/js/pages/game/[game]/modification-comments.tsx_.

All child pages of the game page follow a similar `title` prop format: "Thing - {{gameTitle}}". Let's do something similar here, too.